lib/typescript/class-names-to-type-definition.ts

```diff
@@ -9,6 +9,14 @@ export const EXPORT_TYPES: ExportType[] = ["named", "default"];
 export type QuoteType = "single" | "double";
 export const QUOTE_TYPES: QuoteType[] = ["single", "double"];
 
+export interface ITypeDefinitionOptions {
```

Comment:
The rest of the codebase doesn't use the pattern of prefixing interfaces with `I`. For consistency, let's remove this one.

```suggestion
export interface TypeDefinitionOptions {
```

Reply:
Removed the I.